bin/varnishd/mgt/mgt_vcc.c

```diff
@@ -329,13 +329,13 @@ mgt_vcc_init_vp(struct vcc_priv *vp)
 }
 
 static void
-mgt_vcc_fini_vp(struct vcc_priv *vp, int leave_lib)
+mgt_vcc_fini_vp(struct vcc_priv *vp, int vcc_failed)
 {
 	if (!MGT_DO_DEBUG(DBG_VCL_KEEP)) {
-		VJ_unlink(VSB_data(vp->csrcfile));
-		VJ_unlink(VSB_data(vp->symfile));
-		if (!leave_lib) {
-			VJ_unlink(VSB_data(vp->libfile));
+		VJ_unlink(VSB_data(vp->csrcfile), vcc_failed);
+		VJ_unlink(VSB_data(vp->symfile), vcc_failed);
+		if (vcc_failed) {
+			VJ_unlink(VSB_data(vp->libfile), vcc_failed);
 			VJ_rmdir(VSB_data(vp->dir));
 		}
 	}
```

Comment:
Maybe introduce an enum to avoid magic values for `leave_lib`/`vcc_failed` at call sites?

Reply:
good idea, just added it